src/components/Genomes/Sourmash/History/index.tsx

```diff
@@ -14,16 +14,21 @@ const SourmashHistory: React.FC<HistoryProps> = ({
   return (
     <section>
       <details>
-        <summary>
-          <h4 style={{ display: 'inline', cursor: 'pointer' }}>
-            Search History
-          </h4>
-        </summary>
+        <summary>Search History</summary>
         <div>
-          <p>Here is a list of previous searches in this catalog:</p>
           <div className="genome-search-history">
             {jobs.size === 0 ? (
-              "We can't find any previous search jobs in this browser."
+              <div
+                className="vf-box vf-box-theme--primary vf-box--easy"
+                style={{
+                  backgroundColor: '#d1e3f6',
+                }}
+              >
+                <h3 className="vf-box__heading">
+                  <span className="icon icon-common icon-exclamation-triangle" />{' '}
+                  No previous search jobs found in your browser
```

Comment:
Maybe we can to add some more context here... I suspect most users won't understand that.
```
If you open this website in a different browser or a different computer you won't be able to see the history
```
something like that...